### PR TITLE
Removed unused `elementor-edit-area-preview` class

### PR DESCRIPTION
**Pull request by kobizz**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: assets/dev/js/editor/commands/internal/attach-preview.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ export class AttachPreview extends CommandInternalBaseBase {
 
 			document.$element
 				.addClass( 'elementor-edit-area-active' )
-				.removeClass( 'elementor-edit-area-preview elementor-editor-preview' );
+				.removeClass( 'elementor-editor-preview' );
```
Comment (matipojo) [inline assets/dev/js/editor/commands/internal/attach-preview.js]: We need a method for CSS deprecation.

**Review by matipojo — CHANGES_REQUESTED** [review]

**Comment by arielk** [comment]
@kobizz If it's not resolved any issue I suggest to send it to minor version instead of patch

File: assets/dev/js/editor/commands/internal/attach-preview.js (hunk 1 above)
Comment (kobizz) [reply assets/dev/js/editor/commands/internal/attach-preview.js]: It's impossible

Comment (matipojo) [reply assets/dev/js/editor/commands/internal/attach-preview.js]: A post on dev blog + wait 3 versions is the minimum.

**Pull request closed without merging**